app/index.html

```diff
@@ -40,12 +40,11 @@
                 </main>
             </div>
         </div>
-
-        <div id="bootstrap-app" class="hidden">
+        <!-- @uirouter-refactor add hidden class to bootstrap-app -->
+        <div id="bootstrap-app">
```

Comment:
We might actually want to migrate this to a `core` component similar to rollcall eventually.

Reply:
super. I'm thinking we need at the very least some loading components that trigger while resolve runs, and a trigger between the match and page load too? anyway, I think that's the way to go, yes